src/jit/codegenarm64.cpp

```diff
@@ -3382,6 +3382,77 @@ void CodeGen::genCodeForCompare(GenTreeOp* tree)
     }
 }
 
+//------------------------------------------------------------------------
+// genCodeForJumpCompare: Generates code for jmpCompare statement.
+//
+// A GT_JCMP node is created when a comparison and conditional branch
+// can be executed in a single instruction.
+//
+// Arm64 has a few instructions with this behavior.
+//   - cbz/cbnz -- Compare and branch register zero/not zero
+//   - tbz/tbnz -- Test and branch register bit zero/not zero
+//
+// A GT_JCMP cbz/cbnz node is created when there is a GT_EQ or GT_NE
+// integer/unsigned comparison against #0 which is used by a GT_JTRUE
+// condition jump node.
+//
+// A GT_JCMP cbz/cbnz node is created when there is a GT_TEST_EQ or GT_TEST_NE
```

Comment:
Should read:

`A GT_JCMP tbz/tbnz node ... `

Reply:
You should also mention the different ranges that these instructions support.

```
The cbz/cbnz supports the normal +/- 1MB branch range for conditional branches
The tbz/tbnz supports a  smaller +/- 32KB branch range 
```

Reply:
Done